src/frontend/components/Chat/index.tsx

```diff
@@ -14,6 +14,28 @@ interface ChatProps {
 
 export const Chat = ({ xmpp }: ChatProps) => {
   useEffect(() => {
+    try {
+      // converse.initialize function initialize globally. So everytime this component
+      // is mounted, the initialize function is called and lead sometimes to session lost
+      // from cache and the chat is not displayed.
+      // The workaround here is to add a plugin responsible to close the session everytime
+      // this component is unmounted. But this lead to another issue, the user muste choose
+      // a nickname again.
+      // And because converse is a global object, you can't call plugins.add several times
+      // with the same name... To deal with this issue, calling `converse.plugins.add` is made
+      // in a try/catch block and the error silently fails.
+
+      converse.plugins.add('marsha', {
```

Comment:
There is no method to check if converse has been already initialized or at least if `marsha` plugins has been already added?

Reply:
I didn't found how to do it. I'm talking about that on an issue on converse repo: https://github.com/conversejs/converse.js/issues/1661#issuecomment-829015427

Reply:
You just receive an answer that mentionned the `insertInto` method. You could create a little util that call `initialize` the first time then `insertInto` after ?

Reply:
@jbpenrath I rework the second commit, can you take a look ?